.gitignore

```diff
@@ -8,9 +8,7 @@ __pycache__/
 .DS_Store
 /static/assets/*
 
-# allow files matching sprite-*.svg, index-*.js, index-*.js.map, and index-*.css that are needed for build
+# allow files matching index-*.js, index-*.js.map, and index-*.css that are needed for build
```

Comment:
[dust] Might consider removing the specific filenames from this comment since they're repeated in the code. Like "allow files matching paths specified, needed for build"

That way you don't have to remember to update this code comment (likely someone will forget at some point) when the file list changes in the future